.buildkite/pipelines/quality-gates/pipeline.tests-qa.yaml

```diff
@@ -29,13 +29,16 @@ steps:
         ENVIRONMENT: ${ENVIRONMENT}
       message: "${BUILDKITE_MESSAGE} (triggered by pipeline.tests-qa.yaml)"
 
-  - label: ":rocket: control-plane e2e tests"
-    trigger: "ess-k8s-qa-e2e-tests-daily" # https://buildkite.com/elastic/ess-k8s-qa-e2e-tests-daily
+steps:
```

Comment:
Could you please explain why you introduced another `steps` here? I don't see that change in all the other files and I'm wondering why it has been introduced here.

Reply:
This is a mistake. I am fixing now.

Reply:
This is a mistake. Fixing it now